src/shared_modules/utils/flatbuffers/schemas/syscollectorDeltas/syscollector_deltas.fbs

```diff
@@ -27,6 +27,8 @@ table dbsync_network_address {
     broadcast:string;
     checksum:string;
     item_id:string;
+    metric:string;
```

Comment:
This only happens during deltas, because after that information is sent, it isn't read nor is present in the databases (local and agent). So that info is lost for network address inventory.

Reply:
Thank you!